packages/fms-base/conf/conf.yaml

```diff
@@ -0,0 +1,10 @@
+---
+iteration: ms1
```

Comment:
Further to the comments on setting `--version`, it might make sense to use this field for the timestamp and commit hash, as that is replacing the purpose of using `ms<N>`.

Reply:
Oh, I see what you mean now with timestamp replacing the purpose for that now

Reply:
Wait no, maybe I don't

Reply:
Actually, don't think I've really understood suggestion. Are you imagining `6.0+ms<N>`?

Reply:
What I was suggesting was using the `version` field for the tag and the `iteration` field for the timestamp/commit string. We can drop the `ms<N>` as unnecessary.

You mention elsewhere the potential need to retain a HHMM in the timestamp to ensure ordering if we released multiple on a given day - this would take care of any need to re-issue the same package due to packaging issues as well.

So we'd end up with something like `fms-base_6.0-git202503201505.a1b2c3d4.deb`

Reply:
Ohhh I see, yeah makes sense, nice.